### PR TITLE
Adding Aggregation and signup

### PR DESCRIPTION
**Pull request by khyati-gupta**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -14014,4 +14014,621 @@ Competitor keywords(s) from BirdEye platform can be fetched using following filt
...
+
+<center><b>Example curl request</b></center>
+    
+    curl -H "content-type: application/json" -H "accept:application/json" -X POST -d '{"businessName": "smb_comp","userEmailId": "rhonda@reviewmanagement.com"}' "https://api.birdeye.com/resources/v1/signup/competitor/151134205012586?api_key=92bcd6e0-c102-43fd-8a67-1a7be5258451" -v
+
+<center><b>Example Payload</b></center>
+
+{
+    "businessName": "CompE",
+    "userEmailId": "rhonda@reviewmanagement.com"
```
Comment (ameyavirkar) [inline apiary.apib:14039]: @khyati-gupta do not use client's email address in the documentation.

File: apiary.apib [hunk 2]
```diff
@@ -14014,4 +14014,621 @@ Competitor keywords(s) from BirdEye platform can be fetched using following filt
...
+<center><b>Example Response</b></center>
+
+{
+    "competitorId": 100037739,
+    "templateid": 0,
+    "isAggregationRequired": 1,
+    "type": "Enterprise-Location-Competitor",
+    "businessNumber": 161295237142347,
+    "businessAlreadyPresent": false,
+    "sfdcAccountId": null,
```
Comment (ameyavirkar) [inline apiary.apib:14075]: @khyati-gupta Why does client care of sfdcAccountid and sfdcOpportunityId? these are internal fields for Birdeye. Why do we have them in the response?

File: apiary.apib [hunk 3]
```diff
@@ -14014,4 +14014,621 @@ Competitor keywords(s) from BirdEye platform can be fetched using following filt
...
+
+<center><b>Example curl request</b></center>
+    
+    curl -H "content-type: application/json" -H "accept:application/json" -X POST -d '{"businessName":"Location_compca","userEmailId":"rhonda@reviewmanagement.com","countryCode":"CA","phone":"408-xxx-xxxx","zip":"95035","competitorId":100037693,"type":"Product"}' 
+    "https://api.birdeye.com/resources/v1/signup/competitor/151134205012586?api_key=92bcd6e0-c102-43fd-8a67-1a7be5258451" -v
+
+<center><b>Example Payload</b></center>
+
+    {"businessName": "Location_compca",
+    "userEmailId": "rhonda@reviewmanagement.com",
```
Comment (ameyavirkar) [inline apiary.apib:14217]: @khyati-gupta use a different email address in this payload and the curl commands.

File: apiary.apib [hunk 4]
```diff
@@ -14014,4 +14014,621 @@ Competitor keywords(s) from BirdEye platform can be fetched using following filt
...
+    "countryCode": "CA",
+    "phone": "408-xxx-xxxx",
```
Comment (ameyavirkar) [inline apiary.apib:14219]: @khyati-gupta do not use "xxx" in the documentation. Add some random phone number.

File: apiary.apib [hunk 5]
```diff
@@ -14014,4 +14014,621 @@ Competitor keywords(s) from BirdEye platform can be fetched using following filt
...
+            Accept: application/json
+
++ Response 200
+
+    + Body
+
+{
+    "id": 140179308,
+    "sourceId": 2,
+    "businessId": 100037706,
```
Comment (ameyavirkar) [inline apiary.apib:14604]: @khyati-gupta is this long business number or short business id? we should not return short business id

Comment (khyati-gupta) [reply apiary.apib:14580]: @ameyavirkar : This is short businessId which is getting returned from create competitor API and is being used in creating the enterprise location. Also  as discussed with Sachin Gupta it has been added in GET competitor API as well. 
 If we remove it here then  above APIs will also need code changes as well. Please advise.

**Review by guptas — APPROVED** [review]

**Pull request merged**